src/modules/BootStorage.rb

```diff
@@ -107,9 +107,19 @@ def prep_partitions
     def available_swap_partitions
       ret = {}
 
-      staging.filesystems.select { |f| f.type.is?(:swap) }.each do |swap|
-        blk_device = swap.blk_devices[0]
-        ret[blk_device.name] = blk_device.size.to_i / 1024
+      mounted = Y2Storage::MountPoint.find_by_path(staging, Y2Storage::MountPoint::SWAP_PATH.to_s)
+      if mounted.empty?
+        log.info "No mounted swap found, using fallback."
+        staging.filesystems.select { |f| f.type.is?(:swap) }.each do |swap|
+          blk_device = swap.blk_devices[0]
+          ret[blk_device.name] = blk_device.size.to_i / 1024
```

Comment:
The storage size does not provide any simple method for unit conversions? Like `blk_device.size.to_kiB` ?

Reply:
Well, only in form of `Y2Storage::DiskSize.KiB(blk_device.size)` which is a bit long for me.